src/MainOpt.cxx

```diff
@@ -63,7 +78,10 @@ std::pair<int, std::unique_ptr<MainOpt>> parse_opt(int argc, char **argv) {
   static struct option long_options[] = {
       {"help", no_argument, 0, 'h'},
       {"config-file", required_argument, 0, 0},
+      {"command-uri", required_argument, 0, 0},
+      // Legacy alias for 'command-uri'
```

Comment:
Let's try to avoid taking on legacy baggage already, do you have a really good reason for not breaking backwards compatibility at this point?